### PR TITLE
Gql01 user create ap itest jalalov

### PR DESCRIPTION
**Pull request by Jalalov01**

### CODE REVIEW

File: src/modules/user/_test/userCreateJal.spec.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            const postData = {
+                query: userCreateM,
+                variables: arg,
+            }
+            requestGql(postData)
+                .expect(200)
+                .end((err, res) => {
+                    if (err) return done(err);
+                    const respData = res.body.data
+                    console.log("RESP BODY ===", respData)
```
Comment (AzamatMurzabay) [inline src/modules/user/_test/userCreateJal.spec.js:17]: please remove the console.log

File: src/modules/user/_test/userCreateJal.spec.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            const postData = {
+                query: '',
+                variables: arg,
+            }
+            requestGql(postData)
+                .expect(400)
+                .end((err, res) => {
+                    if (err) return done(err);
+                    const respData = res.body.errors[0]
+                    console.log("RESP BODY ===", respData)
```
Comment (AzamatMurzabay) [inline src/modules/user/_test/userCreateJal.spec.js:36]: please remove the console.log

File: src/modules/user/_test/userCreateJal.spec.js (hunk 1 above)
Comment (AzamatMurzabay) [inline src/modules/user/_test/userCreateJal.spec.js:17]: please remove the console.log

File: src/modules/user/_test/userCreateJal.spec.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+            const postData = {
+                query: userCreateM,
+                variables: userIncorrectType,
+            }
+            requestGql(postData)
+                .expect(400)
+                .end((err, res) => {
+                    if (err) return done(err);
+                    const respData = res.body.errors[0]
+                    console.log("RESP BODY ===", respData)
```
Comment (AzamatMurzabay) [inline src/modules/user/_test/userCreateJal.spec.js:52]: please remove the console.log

File: src/modules/user/_test/userCreateJal.spec.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
+const {expect} = require('chai')
+const {requestGql} = require('../../helper')
+const {arg, userIncorrectType} = require('./data')
+const {userCreateM} = require('./queries')
+describe('USER CREATE', () => {
+    describe('USER CREATE - POSITIVE', () => {
+        it('user create', (done) => {
```
Comment (AzamatMurzabay) [inline src/modules/user/_test/userCreateJal.spec.js:7]: before starting the test cases execution add the 'before hook' example below. 

before('user delete all', (done) => {
            User.deleteMany({});
            return done();
        });

**Review by AzamatMurzabay — COMMENTED** [review]
Hi Akbar! Fantastic work overall! Just a few minor adjustments needed. Thanks!

File: src/modules/user/_test/userCreateJal.spec.js (hunk 4 above)
Comment (Jalalov01) [reply src/modules/user/_test/userCreateJal.spec.js:7]: "Before hook" has been added

File: src/modules/user/_test/userCreateJal.spec.js (hunk 1 above)
Comment (Jalalov01) [reply src/modules/user/_test/userCreateJal.spec.js:17]: Done

File: src/modules/user/_test/userCreateJal.spec.js (hunk 2 above)
Comment (Jalalov01) [reply src/modules/user/_test/userCreateJal.spec.js:36]: Done

File: src/modules/user/_test/userCreateJal.spec.js (hunk 3 above)
Comment (Jalalov01) [reply src/modules/user/_test/userCreateJal.spec.js:52]: Done

**Review by AzamatMurzabay — APPROVED** [review]

**Review by AzamatMurzabay — APPROVED** [review]

**Review by AzamatMurzabay — APPROVED** [review]

**Review by AzamatMurzabay — APPROVED** [review]

**Pull request merged**